internal/pkg/runtime/launch/launcher_linux.go

```diff
@@ -118,7 +118,7 @@ func (l *Launcher) Exec(ctx context.Context, image string, args []string, instan
 			}
 		} else if (l.uid != 0) && (!fakeroot.IsUIDMapped(l.uid) || l.cfg.IgnoreSubuid) {
 			sylog.Infof("User not listed in %v, trying root-mapped namespace", fakeroot.SubUIDFile)
-			l.cfg.Fakeroot = false
+			l.cfg.Fakeroot = true
```

Comment:
This is redundant because it's already inside of an `if` statement where the flag is true.  Just delete the line, although I'm not sure what difference it makes if it is false or true in this case.

Reply:
thanks, updated!